webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js

```diff
@@ -132,4 +134,24 @@ describe('Test utils', () => {
         });
     });
   });
+  
+  it('Check that all test specs belong to a test suites', () => {
+    const testFolders = [];
+    for (const [, value] of Object.entries(constants.SUITES)) {
+      value.forEach(path => testFolders.push(path.split('/')[1]));
+    }
+    const getDirectories =   (src, callback) => {
+      glob(src, callback);
+    };
+    //get all spec files in tests/e2e/default
+    getDirectories('../../../../../../tests/e2e/default/**/*.wdio-spec.js', (err, res)  => {
+      if (err) {
+        console.log('Error', err);
```

Comment:
The test should fail in this case. After you promisified then this test is no longer necessary.